core/src/main/java/org/springframework/security/authentication/TestingAuthenticationToken.java

```diff
@@ -47,7 +47,7 @@ public TestingAuthenticationToken(Object principal, Object credentials, String..
 		this(principal, credentials, AuthorityUtils.createAuthorityList(authorities));
 	}
 
-	public TestingAuthenticationToken(Object principal, Object credentials, List<GrantedAuthority> authorities) {
```

Comment:
In order to reduce confusion around constructs like:

```java
new TestingAuthenticationToken(principal, credentials, AuthorityUtils.createAuthorityList(authorities))
```

I've updated the PR add a new constructor, but also keep this one (instead of deprecate).